src/cli/onefuzz/templates/libfuzzer.py

```diff
@@ -212,6 +229,7 @@ def basic(
         check_fuzzer_help: bool = True,
         expect_crash_on_failure: bool = False,
         minimized_stack_depth: Optional[int] = None,
+        coverage_filter: Optional[str] = None,
```

Comment:
perhaps as a follow-on PR, but can we do some basic validation of these on the CLI?

Reply:
Beyond what we'll get from the change to the `File` type, what did you have in mind?

Reply:
This is a custom-to-onefuzz configuration syntax.  If we could do some sort of validation, say... since it's JSON perhaps parse it with a pydantic model?

Reply:
Captured in #927.